http_stubs/templatetags/stub_tags.py

```diff
@@ -1,18 +1,52 @@
-from typing import Dict
+import json
+from html import unescape
+from json import JSONDecodeError
+from typing import AnyStr, Dict, List
+from urllib import parse
 
 from django import template
+from django.contrib.admin.helpers import Fieldset
+from django.template.defaultfilters import stringfilter
 
 Url = str
 
 register = template.Library()
 
 
 @register.simple_tag(takes_context=True, name='absolute')
-def get_absolute_url_tag(context: Dict, url: Url) -> Url:
+def get_absolute_url_tag(context: Dict, url: Url, fieldset: Fieldset) -> Url:
```

Comment:
Nit: `get_absolute_url_tag` seems too verbose. It consumes 25% of our line length allowance. I suggest one of the variants:
- `get_absolute_url`
- `absolute_url_tag`
- `absolute_url`

That are just as readable but more concise.

Reply:
Changed. Thanks!